lib/src/main/java/org/team100/lib/motor/turning/Falcon6TurningMotor.java

```diff
@@ -40,17 +40,17 @@ public class Falcon6TurningMotor implements Motor100<Angle100> {
      * Friction feedforward in amps, for when the mechanism is stopped, or nearly
      * so.
      */
-    private static final double staticFrictionFFAmps = 0.375;
+    private static final double staticFrictionFFAmps = 0;
```

Comment:
is this a mistake?

Reply:
This is what worked in the testing, I am guessing that since the FF was so hard to tune the PID is just doing most of the work, and we likely will still need to tune the feedfoward.

Reply:
ok, add a TODO

Reply:
done

Reply:
PTAL

Reply:
oh also I moved the feedforward up so the low level feedforward class, so the tuning TODO is there now